cmd/create/cluster/cmd.go

```diff
@@ -1917,6 +1943,26 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	isSharedVPC := false
+	privateHostedZoneID := strings.Trim(args.privateHostedZoneID, " \t")
+	privateHostedZoneRoleARN := strings.Trim(args.privateHostedZoneRoleARN, " \t")
+	baseDomain := strings.Trim(args.baseDomain, " \t")
+	if privateHostedZoneID != "" ||
+		privateHostedZoneRoleARN != "" {
+		if privateHostedZoneID == "" ||
+			privateHostedZoneRoleARN == "" ||
+			baseDomain == "" {
+			r.Reporter.Errorf("To install a cluster into a shared VPC," +
+				"private-hosted-zone-id and private-hosted-zone-role-arn should be provided together")
```

Comment:
```suggestion
				"private-hosted-zone-id, private-hosted-zone-role-arn , and base-domain should be provided together")
```

Reply:
@oriAdler 
Some confused by this comment.
you mentioned above

It is true that baseDomain is required as well, but there is another effort to enable the user to provide baseDomain outside of the scope of the shared VPC. So I think it's ok to check only privateHostedZoneID or privateHostedZoneRoleARN.

so I removed the requirement for base-domain.  Any misunderstanding here?